scripts/bfasst/compare/conformal.py

```diff
@@ -25,14 +26,16 @@ class Conformal_CompareTool(CompareTool):
     LOG_FILE_NAME = "log.txt"
     DO_FILE_NAME = "compare.do"
     GUI_FILE_NAME = "run_conformal_gui.sh"
+    MAPPING_FILE_NAME = "netlist_mapping.py"
+    MAPPED_POINTS_FILE_NAME = "mapped_points.txt"
 
     def __init__(self, cwd, vendor):
         super().__init__(cwd)
 
         assert type(vendor) is flows.Vendor
         self.vendor = vendor
 
-    def compare_netlists(self, design, print_to_stdout=True):
+    def compare_netlists(self, design, mapping_algorithm, print_to_stdout=True):
```

Comment:
See my comments below.  I don't think we want to change the argument list here. Rather, if using the approach described below, you should have access to `self.args` and can use that to get a mapping algorithm option.